test/runtests.jl

```diff
@@ -11,22 +11,22 @@ using Random
 Random.seed!(1234)
 
 @testset "HerbSearch.jl" verbose=true begin
-    include("test_search_procedure.jl")        
-    include("test_context_free_iterators.jl")
-    include("test_sampling.jl")
-    include("test_stochastic/test_stochastic.jl")
-    include("test_genetic.jl")
-    include("test_programiterator_macro.jl")
+    # include("test_search_procedure.jl")        
+    # include("test_context_free_iterators.jl")
+    # include("test_sampling.jl")
+    # include("test_stochastic/test_stochastic.jl")
+    # include("test_genetic.jl")
+    # include("test_programiterator_macro.jl")
```

Comment:
maybe don't commit this file? :) so that the we don't exclude the tests generally in the entire repo?